content/docs/1_guide/19_security/guide.txt

```diff
@@ -31,6 +31,16 @@ Kirby's current version number is stored in `/kirby/composer.json` in the versio
 
 <info>Make sure to never show the version number publicly in one of your templates as that may give attackers valuable information about your site.</info>
 
+### Use versions with security support
+
+Whenever we find or get notified of a vulnerability, we release updates for all Kirby versions with active support and security support. Active support means that the version receives functionality updates, enhancements and bug fixes. Security support means that the version only receives security updates.
+
+Versions that are "end of life" receive no updates whatsoever. We cannot ensure that these versions are free of vulnerabilities and security researchers generally don't review these versions for vulnerabilities either.
```

Comment:
What I'm wondering Shouldn't 3.* versions before a security fix then have an EOL tag as well, instead of 3.* => active-support, at least that might be a bit misleading.

Reply:
I have written this guide section so that it can be linked to from the security issue list in the Panel System view. So users who get sent here are on an older major release that will not receive any more updates. Whereas users who are on 3.x but on a release with a known vulnerability won't get the EOL warning but a specific warning about the vulnerability instead, also with the recommendation to update.

(Of course the update check will only be implemented in 3.x, so it won't be displayed on v1 or v2. But if we ever release v4 and v3 is EOL, the EOL check will become active. All these thoughts are planning ahead.)

The data in the table is structured for the update check logic. I have updated the guide wording though to be more clear. Is it better now?

Reply:
Yes, it's fine!